modules/nw-sriov-configuring-device.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple-networks/configuring-sr-iov.adoc
+
+[id="configuring-sr-iov-devices_{context}"]
+= Configuring SR-IOV network devices
+
+The SR-IOV Network Operator adds the `SriovNetworkNodePolicy.sriovnetwork.openshift.io` Custom Resource Definition (CRD) to {product-title}.
+You can configure the SR-IOV network device by creating a SriovNetworkNodePolicy Custom Resource (CR).
+
+[NOTE]
+=====
+When applying the configuration specified in a SriovNetworkNodePolicy CR, the SR-IOV Operator may drain the nodes, and in some cases, reboot nodes.
+It may take several minutes for a configuration change to apply.
```

Comment:
Add, "Please make sure there are available nodes to handle the evicted workload beforehand."